log4j-mongodb3/src/test/java/org/apache/logging/log4j/mongodb3/MongoDb3Resolver.java

```diff
@@ -0,0 +1,186 @@
+                                    new StatusLoggerStreamProcessor(Level.INFO)))
+                            .error(Processors.named("[" + label + " error]",
+                                    new StatusLoggerStreamProcessor(Level.ERROR)))
+                            .commands(new StatusLoggerStreamProcessor(Level.DEBUG))
+                            .build();
+                case CONSOLE:
+                    return ProcessOutput.namedConsole(label);
+                default:
+            }
+        }
+        throw new NotImplementedException(loggingTarget.toString());
+    }
+
+    public MongoDb3Resolver() {
+        super(MongoClient.class);
+    }
+
+    @Override
+    public void beforeAll(ExtensionContext context) throws Exception {
+        final TestProperties props = TestPropertySource.createProperties(context);
+        final Mongod mongod = Mongod.builder()
+                .processOutput(Derive.given(Name.class)
+                        .state(ProcessOutput.class)
+                        .deriveBy(name -> getProcessOutput(LoggingTarget.getLoggingTarget(LoggingTarget.STATUS_LOGGER),
+                                name.value())))
+                .processConfig(Start.to(ProcessConfig.class)
+                        .initializedWith(ProcessConfig.defaults().withStopTimeoutInMillis(BUILDER_TIMEOUT_MILLIS))
+                        .withTransitionLabel("create default"))
+                // workaround for https://github.com/flapdoodle-oss/de.flapdoodle.embed.mongo/issues/309
+                .packageOfDistribution(new PackageOfCommandDistribution() {
+
+                    @Override
+                    protected Package packageOf(Command command, Distribution distribution,
+                            DistributionBaseUrl baseUrl) {
+                        if (distribution.platform().operatingSystem().type() == OSType.Windows) {
+                            final Package relativePackage = legacyPackageResolverFactory().apply(command)
+                                    .packageFor(distribution);
+                            final FileSet.Builder fileSetBuilder = FileSet.builder()
+                                    .addEntry(FileType.Library, "ssleay32.dll")
+                                    .addEntry(FileType.Library, "libeay32.dll");
+                            relativePackage.fileSet().entries().forEach(fileSetBuilder::addEntries);
+                            return Package.builder()
+                                    .archiveType(relativePackage.archiveType())
+                                    .fileSet(fileSetBuilder.build())
+                                    .url(baseUrl.value() + relativePackage.url())
+                                    .hint(relativePackage.hint())
+                                    .build();
+                        }
+                        return super.packageOf(command, distribution, baseUrl);
+                    }
+                })
```

Comment:
Due to flapdoodle-oss/de.flapdoodle.embed.mongo#309 the Windows tests require a workaround.